probability/windowed_median.cpp

```diff
@@ -0,0 +1,187 @@
+/**
+ * @file
+ * @brief An implementation of a median calculation of a sliding window along a data stream
```

Comment:
Provide a Wikipedia link in Markdown format (if available/possible).
If there's no Wikipedia link, add another web source for algorithm explanation. 🙂